coral-erm/coral
# ILS Connection: use a dropdown list when searching an ILS vendor

This patch allows the user to select an existing ILS vendor from a dropdown list:

![vendors_dropdown](https://user-images.githubusercontent.com/1679997/56038073-4ee63080-5d31-11e9-8651-558be65c9f05.png)

Test plan:
 - enable Coral <-> ILS connection
 - try to create a new Organization
 - check that a dropdown list containing ILS vendors appears
 - check that the ILS vendor informations are fetched
 - check that you can't create the organization if it already exists in Coral
 - check that you can create the organization if it doesn't already exist in Coral
 - disable Coral <-> ILS connection
 - check that the new Organization form behaves as it did before this PR.

If merged, this PR obsoletes PR #551 (issue #550)
